driver/examples/src/test/scala/org/virtuslab/ideprobe/ModulePresent.scala

```diff
@@ -0,0 +1,23 @@
+package org.virtuslab.ideprobe
+
+import org.junit.{Assert, Test}
+
+class ModulePresent extends ModuleTest {
```

Comment:
class name is not very descriptive, Perhaps, we could move the `run` method into the `ModuleTest` class and call it `verifyModulePresent` while at the same time inlining the `testPresentModules` method from this class? As a result we would have one less class with a more descriptive test name